app/controllers/pets_controller.rb

```diff
@@ -5,6 +5,10 @@ class PetsController < ApplicationController
   # GET /pets.json
   def index
     @pets = Pet.all
+
+    if not @pets.nil?
+      @pets = @pets.sort { |x, y| -(x.submitted_at <=> y.submitted_at) }
```

Comment:
Another issue having a sort algorithm applied inplace. 
Imagine a situation when you need to export a list of pets in a file, with exact the same order as on the page. 
You will have to copy the algorithm. That points out that the code can be extracted as s method, and that it is not appropriate to a controller. It can be placed to the model, or service object. 
--
In other words, ask yourself: is it possible i need to reuse this logic? How would i do this?